src/fqe/fqe_decorators.py

```diff
@@ -261,7 +262,20 @@ def fermionops_tomatrix(ops: 'FermionOperator', norb: int) -> numpy.ndarray:
 
         tensor[tuple(index_mask)] += (-1)**parity * ops.terms[term]
 
-    return tensor
+    tensor2 = numpy.zeros_like(tensor)
+    length = 0
+    seed = []
+    for i in range(rank // 2):
+        seed.append(i)
```

Comment:
```suggestion
    seed = list(range(rank // 2))
```

Reply:
I've just updated (this does not have to be list) see below. Would be indeed great if you could add a test.